orca_detector/generator.py

```diff
@@ -1,94 +1,180 @@
+"""
+Keras custom generators for the Orca project.
+
+W251 (Summer 2019) - Spyros Garyfallos, Ram Iyer, Mike Winton
+
+Adapted from https://github.com/DTaoo/VGGish
+"""
+
 import keras
 import librosa
 import numpy as np
+import pprint
```

Comment:
Which linter are you using?

Reply:
This is intentional: pprint = "pretty print".  It allows you to print more nicely formatted lists and arrays.

Reply:
I was referring to the imports alphabetizing order, I thought I'd done that